test/kixi/gwyn/gwyn_test.clj

```diff
@@ -10,11 +10,19 @@
   {:lfb-historic-incidents ["data/lfb_historical_fire_non_residential.csv"
                             sc/LfbHistoricIncidents]})
 
+(def fire-stations-data
+  {:fire-station-lookup-table ["data/fire_station_data.csv" sc/FireStations]})
+
 (def test-data
   (reduce merge (map (fn [[k [p s]]]
                        (hash-map k (tu/csv-to-dataset p s)))
                      data-info)))
 
+(def fire-stations
+  (reduce merge (map (fn [[k [p s]]]
```

Comment:
it feels like doing a reduce assoc would be clearer here?
```
(reduce (fn [acc [k [p s]]] (assoc acc k (tu/csv-to-dataset p s))) fire-stations-data)
```
unless I misunderstand